meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -169,7 +169,7 @@ IMAGE_CMD_resin-sdcard () {
 	# Create a vfat filesystem with boot files
 	BOOT_BLOCKS=$(LC_ALL=C parted -s ${RESIN_SDIMG} unit b print | awk '/ 1 / { print substr($4, 1, length($4 -1)) / 512 /2 }')
 	rm -rf ${WORKDIR}/boot.img
-	mkfs.vfat -n "${RESIN_BOOT_FS_LABEL}" -S 512 -C ${WORKDIR}/boot.img $BOOT_BLOCKS
+	mkfs.vfat -n "${RESIN_BOOT_FS_LABEL}" -F 32 -S 512 -C ${WORKDIR}/boot.img $BOOT_BLOCKS
```

Comment:
Because the idea of this change is to sync partition type with file-system type. While file-system type can be relaxed, partition one can't.